elkserver/docker/redelk-base/redelkinstalldata/scripts/modules/alarm_dummy/module.py

```diff
@@ -40,7 +40,7 @@ def run(self):
         return(ret)
 
     def alarm_dummy(self):
-        q = "c2.log.type:ioc AND NOT tags:%s"%(info['submodule'])
+        q = "c2.log.type:ioc AND NOT tags:alarm_*"
```

Comment:
I would keep the submodule name so it will alert for everything that has not been alerted by `alarm_dummy` before (to keep the behavior consistent with other alarms.

Reply:
makes sense.